pages/CreatingAnExploration.html

```diff
@@ -9,20 +9,22 @@
 
   <p>
     After you've played an exploration, why not try your hand at making one? To
-    do so, navigate to the gallery, and look for the "Create Exploration"
+    do so, navigate to the gallery, and look for the "Create"
     button in the top navbar.
   </p>
 
   <img src="static/images/galleryCreateButton.png" style="width: 100%;">
```

Comment:
In this image it may be a good idea to draw an oval (maybe with an arrow pointing to it) around the create button in the navbar. (Make this look as nice/clean as possible.)

Also, could you use the default library page instead, which is what you get on direct navigation to /library?